parser/quakec-parser.js

```diff
@@ -1,6 +1,56 @@
 const Lexer = require("./quakec-lexer").Lexer;
 const Range = require("./quakec-common").Range;
 
+/** @typedef {"binarySubtractRequiresLeadingWhitespace"} LanguageFeatures */
+
+class FeatureInfo {
+    constructor() {
+        this.binarySubtractRequiresLeadingWhitespace = false;
+    }
+}
+
+/** 
+ * Feature handler callback.
+ * @callback FeatureHandler
+ * @param {boolean} supported Whether the support check passed or not.
+ */
+
+class Feature {
+    /**
+     * Call a handler function. 
```

Comment:
This is the main method of the Feature namespace. Update the comment to something like:

> Check if the given language feature is supported. An optional callback can be provided for custom language feature support handling and error messaging.